src/Microsoft.CSharp/dir.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
+    <PackageVersion>4.4.1</PackageVersion>
     <AssemblyVersion>4.0.3.0</AssemblyVersion>
```

Comment:
We would need to double check but if you changed any of the .NET Framework implementations you would also need to bump the fourth part of the assembly versions.

Reply:
The changes in Microsoft.CSharp restore some of the original behavior of Desktop (serialization attribute & ctor). I see that the package compiles against netstandard but I don't know if the Desktop framework uses it?...

Reply:
For MIcrosoft.Csharp I'm it isn't used on desktop, but my comment was meant to be a general question for all the libraries here. We essentially need to see if there are placeholders or facades in the desktop folders for these packages.

Reply:
I browsed through the list of packages, all of them are partical facades for netfx. There can't be placeholders for any of these assemblies because otherwise none of the exceptions in it would have been serializable (the intent of this change). right?

Reply:
any package that has the same identity, ex Microsoft.CSharp, should have a placeholder in the package (i.e. a file named `_._`) in the package.

Reply:
I just noticed I didn't follow up on that. Not sure, is there any action needed here?

Reply:
Please just double check all the packages to ensure we didn't update desktop assets.